src/Context/UserContext.js

```diff
@@ -0,0 +1,5 @@
+import React from "react"
+
+const UserContext = React.createContext()
+
```

Comment:
So then does the data you pass in from the app query get added here? Nice!

Reply:
Yes!  The data is being added in app.js.  Havent figured out how to change or update context yet.  Going to look into adding more of the methods that we are passing around a lot to useContext.